golang.req

```diff
@@ -21,6 +33,9 @@ end
 requires = []
 
 unless filelist.empty? then
+
+	requires << "golang(API) = " + go_get_version()
```

Comment:
@flavio

Yes...it will drag requirement of go itself...I know this commit may be not "fine"...but actually I just want to pull the SLE_11_SP3 fix...I just didn't know how to send a single commit separately...and, this commit is meaningful to me because in my branch I was implementing "buildmode=shared" and "buildmode=c-shared" and "linkshared" support. In that case, the dependency of go itself is useful...because the shared standard libraries (those .shlibname) are in go itself. But my work just hang there...because there's a bug happened in our go (I'm not sure whether it's packaging or upstream), about which I sent an email to opensuse-go mailing list some days ago.

I promised on the mailing list that I'll require go itself only for shared built stuff. But since my shared build support is not finished. I certianlly don't know how to set these codes conditional.

And please reply to #4 when you're free...because I really didn't find the "strong evidences" that can prevent me from shipping prebuilt ".a" stuff...everyone is telling me "hey! people just use .go!", I know that. But logically it doesn't explain why I shouldn't use ".a". And I found myself a lot of "evidences" that ".a" could be used. we did that because we could do...and I think most of the people didn't think human-being can live on Mars, but that is what NASA is trying to do...:-D